frontend/tests/playwright/product/product_image.spec.ts

```diff
@@ -17,17 +12,39 @@ test.describe('Product image test', () => {
 
    test('product with multiple images ', async ({ page }) => {
       await page.goto('/products/repair-business-toolkit');
+      const viewPort = page.viewportSize();
 
       const firstImageSrc = await page
          .locator('.swiper-slide.swiper-slide-active')
          .getByRole('img')
          .first()
          .getAttribute('src');
-      const firstImage = page.locator(`img[src="${firstImageSrc}"]`).first();
+
+      const firstImage =
+         viewPort!.width > 1000
```

Comment:
I am curious why we are using `1000` for the breakpoint between `mobile` and `desktop`. Isn't it usually around `760`?

Reply:
Updated in https://github.com/iFixit/react-commerce/pull/1244/commits/d8a630bf635d7b2ca38b7e7b509ffa9754776138.